source/best-practices/backup.rst

```diff
@@ -80,6 +80,11 @@ rsync
    # 执行命令开始备份
    $ backup.sh
 
+.. admonition:: 免责声明
+
+   本脚本完全基于编写者在自己的计算机上备份文件的经验总结，并不具备普遍适用性，
+   仅供读者参考。
+
```

Comment:
Better to move the disclaimer into the script.

Reply:
Done in 51a1439.